ports/stm32f4/common-hal/busio/SPI.c

```diff
@@ -383,20 +383,20 @@ void common_hal_busio_spi_unlock(busio_spi_obj_t *self) {
 
 bool common_hal_busio_spi_write(busio_spi_obj_t *self,
         const uint8_t *data, size_t len) {
-    HAL_StatusTypeDef result = HAL_SPI_Transmit (&self->handle, (uint8_t *)data, (uint16_t)len, 2);
+    HAL_StatusTypeDef result = HAL_SPI_Transmit (&self->handle, (uint8_t *)data, (uint16_t)len, 500);
     return result == HAL_OK ? 1 : 0;
```

Comment:
```suggestion
    return result == HAL_OK;
```

No need for the ternaries here and below because the equality comparison returns a bool already.